README.md

```diff
@@ -2,49 +2,17 @@
 
 [![CircleCI](https://circleci.com/gh/rage/quizzes/tree/master.svg?style=svg)](https://circleci.com/gh/rage/quizzes/tree/master)
 
-You can set up you shell to automatically switch node version between folders. [Instructions](https://github.com/nvm-sh/nvm#deeper-shell-integration).
```

Comment:
I think we should keep the original setup instructions, too — just under a different subsection. We could first have the new setup instructions under the title "Setup with docker-compose", and then the original setup instructions under the title "Manual setup".